app/about.js

```diff
@@ -49,6 +49,8 @@ ipcRenderer.on('debugInfo', (event, reference, timeleft, settingsfile) => {
     timeleftElement.innerHTML = timeleft
     let settingsfileElement = document.getElementById('settingsfile')
     settingsfileElement.innerHTML = settingsfile
+    let dndElement = document.getElementById('dndElement')
+    dndElement.innerHTML = getDnD
```

Comment:
Think about better name for `getDND`. That's how you could name the function to get DND state. Check others ie `timeleft` or `settingsfile`: it clearly says what it is.